tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-inference-viewer.html

```diff
@@ -87,30 +90,33 @@
     <div class="row header">
       <div class="marker"></div>
       <div class="run">Run</div>
+      <div class="model">Model</div>
```

Comment:
make model column only appear when there are two models (see how the existing label class column does this for classification vs regression models using "class$")